examples/aerospike-cluster-values.yaml

```
@@ -0,0 +1,103 @@
+## Aerospike cluster size
+replicas: 3
+
+## Aerospike Database Docker image
+image:
+  repository: aerospike/aerospike-server-enterprise
+  tag: 7.1.0.0
+
+## Aerospike access control configuration
+aerospikeAccessControl:
+  users:
+    - name: admin
+      secretName: auth-secret
+      roles:
+        - sys-admin
+        - user-admin
```

```suggestion
        - user-admin
        - read-write
```


Enables read-write access to the user.

---

done, please review